module/zcommon/zpool_prop.c

```diff
@@ -133,6 +136,9 @@ zpool_prop_init(void)
 	zprop_register_number(ZPOOL_PROP_ASHIFT, "ashift", 0, PROP_DEFAULT,
 	    ZFS_TYPE_POOL, "<ashift, 9-16, or 0=default>", "ASHIFT", B_FALSE,
 	    sfeatures);
+	zprop_register_number(ZPOOL_PROP_DEDUP_TABLE_QUOTA, "dedup_table_quota",
+	    0, PROP_DEFAULT, ZFS_TYPE_POOL, "<size>", "DDTQUOTA", B_FALSE,
+	    sfeatures);
```

Comment:
Wouldn't default to auto make more sense than disabled?

Reply:
yes, fixed